bin/interpret/analyzefiles.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/python
 import os,sys,inspect
 import numpy as np
-
+from scipy import stats
```

Comment:
where is this module needed in this script?

Reply:
ohhh hmm I think I tried adding that in to calculate standard error instead of standard deviation (consistent with what you used in the paper), but then it didn't work, and I forgot to remove this line.

Reply:
no worries! If you prefer to change the calculation to standard error that's fine with me too.
You can always do it the old school way with np.std(x) / (float(len(x))**0.5)

Reply:
I think i actually tried that first before trying a new package and it didn't work because of the way numpy sets up the data structure, so I'm good with sticking to standard deviation